modules/roary/main.nf

```diff
@@ -0,0 +1,43 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process ROARY {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::roary=3.13.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/roary:3.13.0--pl526h516909a_0"
+    } else {
+        container "quay.io/biocontainers/roary:3.13.0--pl526h516909a_0"
+    }
+
+    input:
+    tuple val(meta), path(gff)
+
+    output:
+    tuple val(meta), path("results/*"), emit: results
+    tuple val(meta), path("results/core_gene_alignment.aln"), emit: aln, optional: true
+    path "versions.yml"               , emit: versions
```

Comment:
```suggestion
    tuple val(meta), path("results/*")                    , emit: results
    tuple val(meta), path("results/*.aln"), optional: true, emit: aln
    path "versions.yml"                                   , emit: versions
```

Reply:
Would that simpler glob work?

Reply:
This should work, because only one `*.aln` should be created. I'm not aware of alternate multiple sequence alignments being created